catalog-rest-service/src/main/resources/json/schema/entity/services/connections/pipeline/dagsterConnection.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/pipeline/dagsterConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "DagsterConnection",
+  "description": "Dagster Metadata Database Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.services.connections.pipeline.DagsterConnection",
+  "definitions": {
+    "DagsterType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": ["Dagster"],
+      "default": "Dagster"
+    }
+  },
+  "properties": {
+    "type": {
+      "title": "Service Type",
+      "description": "Service Type",
+      "$ref": "#/definitions/DagsterType",
+      "default": "Dagster"
+    },
+    "hostPort": {
+      "title": "Host And Port",
+      "description": "Pipeline Service Management/UI URI.",
+      "type": "string",
+      "format": "uri"
+    },
+    "numberOfStatus": {
```

Comment:
Could you please open a ticket to eventually move the `numberOfStatus` to the `sourceConfig`? Might be cleaner to handle it there rather than in each pipeline config. Thanks!